docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx

```diff
@@ -259,39 +268,45 @@ class CustomTask(CustomTaskBase):
       self._as.publish_feedback(self._feedback)
 
     return True
-```
 
-This sample custom task iterates through the two available data variables and outputs them to the feedback topic. If neither of the 
-variables have any data in them the task is aborted.
+def __create_task__():
+	return InputLooperTask()
+```
 
 ##### Custom Task Class
 
+To leverage custom tasks users will need to create a new file that defines a new class which inherits from `CustomTaskBase`. The `CustomTaskBase` class is 
+found in the `onav_Itasks.custom_task_base` package. It exposes a `SimpleActionServer` (see <a href="http://wiki.ros.org/actionlib_tutorials/Tutorials/Writing%20a%20Simple%20Action%20Server%20using%20the%20Execute%20Callback%20%28Python%29">here</a> 
+for further details) object as `_as`, a `UITaskFeedback` object as `_feedback` and a `UITaskResults` object as `_result` to be used as part of 
+the tasks functionality. The custom task manager requires the user to also define a `run_task` method that takes a `UITaskGoal` object as a parameter.
+
 ```py 
-from onav_custom_tasks.custom_task_base import *
+from onav_tasks.custom_task_base import *
 
 class CustomTask(CustomTaskBase):
   def __init__(self):
-      super().__init__("log_inputs")
+      super().__init__("input_looper")
 
   def run_task(self, goal):
 ```
 
-To leverage custom tasks users will need to create a new file that defines a new class called `CustomTask` which inherits from 
-`CustomTaskBase`. The `CustomTaskBase` class is found in the `onav_custom_tasks.custom_task_base` package. It exposes a `SimpleActionServer` 
-object as `_as`, a `UITaskFeedback` object as `_feedback` and a `UITaskResults` object as `_result` to be used as part of the tasks functionality. The custom 
-task manager requires the user to also define a `run_task` method that takes a `UITaskGoal` object as a parameter.
-
 ##### Aborting Tasks 
+
+When running the task users may handle errors through the action servers `set_aborted()` function.  When this function is called the entire mission 
+will be aborted. In this sample the task is aborted if neither of the lists have any values.
+
 ```py 
-if len(goal.strings) == 0 or len(goal.floats) == 0:
+if len(goal.strings) == 0 and len(goal.floats) == 0:
+  # Task and running mission will be aborted in this case
   self._as.set_aborted()
   return False
 ```
 
-When running the task users may handle errors through the action servers `set_aborted()` function.  When this function is called the entire mission 
-will be aborted. In this sample the task is aborted if neither of the lists have any values.
-
 ##### Setting feedback
+
+Users can set the feedback state using the `SimpleActionServer` class's standard methods. In this case the task iterates through the comma deliminated
+strings and floats and writes them to the feedback state before publishing them.  Upon success the `run_task` method returns True.

```

Comment:
What happens if we return False? Does the entire mission get aborted?

Reply:
No in fact it proceeds as normal. I will adjust the documentation to reflect this. I think it's a good idea to leave it as such so that users can leverage the false as a sort of "Failed but proceed" but it will require clear documentation.